src/Command/ProbeDispatcherCommand.php

```diff
@@ -284,95 +280,66 @@ public function sendInstruction(array $instruction, $expectedRuntime = null): vo
-                break;
-
             case 'probe':
-                if (200 === $status) {
-                    $cleaned = [];
-
-                    foreach ($contents as $id => $content) {
-                        if (!isset($content['type'], $content['timestamp'], $content['targets'])) {
-                            // TODO: Good warning
-                            $this->logger->warning("Response ($status) from worker $pid is missing either a type, timestamp or targets key.");
-                        } else {
-                            $cleaned[$id] = $content;
-                        }
-                    }
-
-                    $this->logger->info("Enqueueing the response from worker $pid.");
+                $this->logger->info("dispatcher: enqueueing the response from worker $pid.");
 
-                    $items = $this->expandProbeResult($cleaned);
-                    foreach ($items as $key => $item) {
-                        $queue = $this->queues[$key % $this->numberOfQueues];
-                        $queue->enqueue($item);
-                    }
-                } else {
-                    $this->logger->error("Response ($status) from worker $pid unexpected.");
+                $items = $this->expandProbeResult($contents);
+                foreach ($items as $key => $item) {
+                    $queue = $this->queues[$key % $this->numberOfQueues];
+                    $queue->enqueue($item);
                 }
                 break;
 
             case GetConfigHttpWorkerCommand::class:
+                $this->logger->info('dispatcher: started handling new configuration response');
                 if (200 === $status) {
+                    $this->logger->info('dispatcher: applying new configuration');
                     $etag = $response['headers']['etag'];
                     $this->configuration->updateConfig($contents, $etag);
-                    $this->logger->info("Response ($status) from worker $pid config applied (".$this->configuration->getAllProbesDeviceCount().' devices)');
+                    $this->logger->info(sprintf('dispatcher: new configuration applied (took %.2f seconds)', microtime(true) - $startAt));
+                    $this->logger->info(sprintf('dispatcher: new configuration has %d probes and %d devices', count($this->configuration->getProbes()), $this->configuration->getAllProbesDeviceCount()));
 
                     $count = 0;
                     foreach ($this->configuration->getProbes() as $probe) {
                         $count += ceil($this->configuration->getProbeDeviceCount($probe->getId()) / $this->devicesPerWorker);
                     }
                     $this->workerManager->setNumberOfProbeProcesses(intval($count));
                 } else {
-                    $this->logger->info("Response ($status) from worker $pid received");
-                }
-                break;
-
-            case PostStatsHttpWorkerCommand::class:
```

Comment:
This is nice to know ?

Reply:
Should move this logging to the PostStatsHttpWorker instead. I can add some extra logging there?

Reply:
Sounds good